File: Examples/Objective-C/Remote Control/Remote Control/AdminViewTabBarController.m
//
//  AdminViewBarViewController.m
//  Remote Control
//
//  Created by Calvin on 7/7/17.
//  Copyright © 2017 Bridgefy Inc. All rights reserved.
//

#import "AdminViewTabBarController.h"

@interface AdminViewTabBarController () <UITabBarDelegate>

@end

@implementation AdminViewTabBarController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    self.title = @"Admin panel";
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (void)sendObject:(id)object sender:(id)sender {
    [self.mvc sendObject:object sender:sender];
}

@end

File: Examples/Swift/Remote Control/Remote Control/AdminViewTabBarController.swift
//
//  AdminViewTabBarController.swift
//  Remote Control
//
//  Created by Calvin on 7/11/17.
//  Copyright © 2017 Bridgefy Inc. All rights reserved.
//

import UIKit

class AdminViewTabBarController: UITabBarController {
    
    public var mvc: MainViewController?

    override func viewDidLoad() {
        super.viewDidLoad()
        
        self.title = "Admin panel"
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    public func send(object: Any?, sender: Any) {
        if (self.mvc != nil) {
            self.mvc!.send(object: object, sender: sender);
        }
    }
    
}
